server/sonar-server/src/main/java/org/sonar/server/batch/Messages.java

```diff
@@ -19,8 +19,10 @@
  */
 package org.sonar.server.batch;
 
-public interface Messages {
-
-  String NO_PERMISSION = "You're not authorized to execute any SonarQube analysis. Please contact your SonarQube administrator.";
+public class Messages {
+  private Messages() {
+    // constants
+  }
 
+  static String NO_PERMISSION = "You're not authorized to execute any SonarQube analysis. Please contact your SonarQube administrator.";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "NO_PERMISSION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS3008)